Source/Stationeers.Addons/Modules/Workshop/WorkshopManagerPatch.cs

```diff
@@ -0,0 +1,119 @@
+﻿using Assets.Scripts;
+using Assets.Scripts.Steam;
+using HarmonyLib;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using UnityEngine;
+
+namespace Stationeers.Addons.Modules.Workshop
+{
+    public class WorkshopManagerPatch
+    {
+        // Why does System.IO not have a directory copy method?
+        // Stolen from here: https://stackoverflow.com/questions/1974019/folder-copy-in-c-sharp
+        private static void copyDirectory(string strSource, string strDestination)
+        {
+            if (!Directory.Exists(strDestination))
+            {
+                Directory.CreateDirectory(strDestination);
+            }
+
+            DirectoryInfo dirInfo = new DirectoryInfo(strSource);
```

Comment:
You can safely change the variable declaration to `var dirInfo = ...` to stay consistent with the whole code base.
Please, apply this to all changed files.